main/abuild/0001-abuild-properly-detect-package-size-on-btrfs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/abuild.in b/abuild.in
+index 14955357af..38d2598b8f 100644
+--- a/abuild.in
++++ b/abuild.in
+@@ -812,7 +812,7 @@ prepare_metafiles() {
+ 	cd "$dir"
+ 	mkdir -p "$controldir"
+ 	local builddate=$(date -u "+%s")
+-	local size=$(du -sk | awk '{print $1 * 1024}')
++	local size=$(sync;du -sk | awk '{print $1 * 1024}')
```

Comment:
@fabled I might have a solution, currently testing 3 filesystems which seems to be affected
- BTRFS
- ZFS
- ecryptfs